spec/helpers/application_helper_spec.rb

```diff
@@ -48,6 +48,7 @@
     it "includes SiteConfig.admin_action_taken_at" do
       Timecop.freeze do
         allow(SiteConfig).to receive(:admin_action_taken_at).and_return(5.minutes.ago)
+        allow(ApplicationConfig).to receive(:[]).with("RELEASE_FOOTPRINT").and_return("abc123")
```

Comment:
This test was failing because it seems like ApplicationConfig["RELEASE_FOOTPRINT"] was "", hence it was returning early in the test based on this code 

```
    release_footprint = ApplicationConfig["RELEASE_FOOTPRINT"]
    return path if release_footprint.blank?

    "#{path}-#{params[:locale]}-#{release_footprint}-#{SiteConfig.admin_action_taken_at.rfc3339}"
```

I've reset the value in the test again. I went through my code and I'm not totally sure why this failed in this PR as I'm not resetting the value of RELEASE_FOOTPRINT anywhere in my tests, however it seems very coincidental. 

Whilst this is a fix for the test, I wanted a second pair of 👀's before merging in case I've overlooked something.  

cc @rhymes

Reply:
@Ridhwana I think it was failing because of `config/initializers/set_release_footprint.rb` which is initialized to `HEROKU_SLUG_COMMIT` on startup? That's the only clue I have :D